src/packages/@workspaces/features/rest-explorer/components/http-url-section/HttpUrlSection.svelte

```diff
@@ -179,13 +181,22 @@
       onUpdateRequestState({ requestSplitterDirection: e.detail });
     }}
   /> -->
-  <Tooltip title={"Save"} placement={"bottom"} distance={12} zIndex={10}>
+  <Tooltip
+    title={isGuestUser ? "Login to save" : "Save"}
+    show={requestUrl !== ""}
+    placement={"bottom-left"}
```

Comment:
guest user can also save, and I think requestUrl has not any dependency to show or hide tooltip
cc: @nagesh953 @Monika-APM @astitva-techdome